GliaWidgets/Sources/QuickLookBased/QuickLookBased.Implementation.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import UIKit
+import QuickLook
+
+extension QuickLookBased.ThumbnailGenerator {
+    static func create(uiScreen: UIKitBased.UIScreen) -> Self {
+        if #available(iOS 13.0, *) {
```

Comment:
Will we need this check if this change will be applied once deployment target is iOS 13?

Reply:
No, it will be removed once deployment target is changed.